jupyterhub/services/auth.py

```diff
@@ -138,6 +138,27 @@ class HubAuth(Configurable):
     def _cookie_cache(self):
         return _ExpiringDict(self.cookie_cache_max_age)
 
+    def _check_response_for_authorization(self, r):
+        """Verify the response for authorizing a user
+
+        Raises an error if the response failed, otherwise returns the json
+        """
+        if r.status_code == 404:
+            app_log.warning("No Hub user identified for request")
+            data = None
+        elif r.status_code == 403:
+            app_log.error("I don't have permission to verify cookies, my auth token may have expired: [%i] %s", r.status_code, r.reason)
```

Comment:
Update message to be more general:

"I don't have permission to check authorization with JupyterHub; my auth..."